pkgs/applications/editors/quartus-prime/default.nix

```diff
@@ -0,0 +1,124 @@
+{ buildFHSUserEnv, makeDesktopItem, stdenv, lib, requireFile, unstick, cycloneVSupport ? true }:
+
+let
+  quartus = stdenv.mkDerivation rec {
+    version = "19.1.0.670";
+    pname = "quartus-prime";
+
+    src = let
+      require = {name, sha256}: requireFile {
+        inherit name sha256;
+
+        message = ''
+          This nix expression requires that ${name} is already part of the store.
+          Download it from ${meta.homepage} and add it to the nix store with:
+
+              nix-prefetch-url <URL>
```

Comment:
BTW I'd suggest adding here that it can also be done via `nix-store --add <file>`

Reply:
Sure, is there any reason for preferring one over the other? Going to hold off on making any changes until we get the hash issue sorted out though.

Reply:
I've removed the custom message. The default message now suggests `nix-store --add-fixed sha256` or `nix-prefetch-url --type sha256 file://path/to`. C